gum/backend-qnx/gummemory-qnx.c

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (C) 2008-2011 Ole André Vadla Ravnås <ole.andre.ravnas@tillitech.com>
+ *
+ * Licence: wxWindows Library Licence, Version 3.1
+ */
+
+#include "gummemory.h"
+#include "gumqnx-priv.h"
+
+#include <gio/gio.h>
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <sys/procfs.h>
+#include <stdio.h>
```

Comment:
Regarding includes, we tend to use the following pattern:

[include your own header]

[include internal header(s)]

[include system/library headers]

Entries in each section should be alphabetically sorted.